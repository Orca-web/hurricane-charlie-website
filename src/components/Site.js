import React, { Component } from 'react';
import Header from './Header';
import Content from './Content';
import Footer from './Footer';

class Site extends Component {
  constructor() {
    super()
    this.state = { gallery: [] };
  }

  render() {
    return (
      <div>
        <Header nav={this.state.nav}/>
        <Content />
        <Footer />
      </div>
    ) 
  }
}

export default Site