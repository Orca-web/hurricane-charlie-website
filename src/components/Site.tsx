import React, { Fragment } from 'react';
import { BrowserRouter } from 'react-router-dom';

import Header from './Header';
import Content from './Content';
import Footer from './Footer';

const Site = () => (
    <BrowserRouter>
        <Fragment>
            <Header />
            <Content />
            <Footer />
        </Fragment>
    </BrowserRouter>
);

export default Site;