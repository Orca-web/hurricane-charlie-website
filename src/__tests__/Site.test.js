import React from 'react';
import { shallow } from 'enzyme';
import Site from '../components/Site';

const app = shallow(<Site />);

describe('Site', () => {
  it('renders correctly', () => {
    expect(app).toMatchSnapshot();
  });
});