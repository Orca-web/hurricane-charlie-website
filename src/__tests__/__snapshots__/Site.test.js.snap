// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Site renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Site />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Header
          navClick={[Function]}
          navData={
            Array [
              Object {
                "active": false,
                "id": 0,
                "name": "gallery",
              },
              Object {
                "active": false,
                "id": 1,
                "name": "contact",
              },
              Object {
                "active": true,
                "id": 2,
                "name": "shop",
              },
            ]
          }
        />,
        <Content
          activeContent="shop"
          galleryData={
            Array [
              Object {
                "id": 1,
                "name": "leaf insect",
                "size": "large",
                "src": "thumbs-001.jpg",
              },
              Object {
                "id": 2,
                "name": "desert eyes",
                "size": "wide",
                "src": "thumbs-002.jpg",
              },
              Object {
                "id": 3,
                "name": "bardo pond",
                "size": "wide",
                "src": "thumbs-003.jpg",
              },
              Object {
                "id": 4,
                "name": "angled pike",
                "size": "small",
                "src": "thumbs-004.jpg",
              },
              Object {
                "id": 5,
                "name": "mission of burma",
                "size": "small",
                "src": "thumbs-005.jpg",
              },
              Object {
                "id": 6,
                "name": "bass overload",
                "size": "tall",
                "src": "thumbs-006.jpg",
              },
              Object {
                "id": 7,
                "name": "folläkzoid",
                "size": "tall",
                "src": "thumbs-007.jpg",
              },
              Object {
                "id": 8,
                "name": "robot workshop",
                "size": "large",
                "src": "thumbs-008.jpg",
              },
              Object {
                "id": 9,
                "name": "angler fish",
                "size": "wide",
                "src": "thumbs-009.jpg",
              },
              Object {
                "id": 10,
                "name": "geometric bug",
                "size": "wide",
                "src": "thumbs-010.jpg",
              },
              Object {
                "id": 11,
                "name": "you mug",
                "size": "small",
                "src": "thumbs-011.jpg",
              },
              Object {
                "id": 12,
                "name": "mutant bmx",
                "size": "small",
                "src": "thumbs-012.jpg",
              },
              Object {
                "id": 13,
                "name": "nf fly",
                "size": "tall",
                "src": "thumbs-013.jpg",
              },
              Object {
                "id": 14,
                "name": "desert smoker",
                "size": "tall",
                "src": "thumbs-014.jpg",
              },
              Object {
                "id": 15,
                "name": "geometric butterfly",
                "size": "large",
                "src": "thumbs-015.jpg",
              },
              Object {
                "id": 16,
                "name": "dylan carlson",
                "size": "wide",
                "src": "thumbs-016.jpg",
              },
              Object {
                "id": 17,
                "name": "octopsss",
                "size": "wide",
                "src": "thumbs-017.jpg",
              },
              Object {
                "id": 18,
                "name": "collapse",
                "size": "small",
                "src": "thumbs-018.jpg",
              },
              Object {
                "id": 19,
                "name": "rat race",
                "size": "small",
                "src": "thumbs-019.jpg",
              },
              Object {
                "id": 20,
                "name": "papa m house",
                "size": "tall",
                "src": "thumbs-020.jpg",
              },
            ]
          }
        />,
        null,
        <Footer
          navClick={[Function]}
          navData={
            Array [
              Object {
                "active": false,
                "id": 0,
                "name": "gallery",
              },
              Object {
                "active": false,
                "id": 1,
                "name": "contact",
              },
              Object {
                "active": true,
                "id": 2,
                "name": "shop",
              },
            ]
          }
        />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "navClick": [Function],
          "navData": Array [
            Object {
              "active": false,
              "id": 0,
              "name": "gallery",
            },
            Object {
              "active": false,
              "id": 1,
              "name": "contact",
            },
            Object {
              "active": true,
              "id": 2,
              "name": "shop",
            },
          ],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "activeContent": "shop",
          "galleryData": Array [
            Object {
              "id": 1,
              "name": "leaf insect",
              "size": "large",
              "src": "thumbs-001.jpg",
            },
            Object {
              "id": 2,
              "name": "desert eyes",
              "size": "wide",
              "src": "thumbs-002.jpg",
            },
            Object {
              "id": 3,
              "name": "bardo pond",
              "size": "wide",
              "src": "thumbs-003.jpg",
            },
            Object {
              "id": 4,
              "name": "angled pike",
              "size": "small",
              "src": "thumbs-004.jpg",
            },
            Object {
              "id": 5,
              "name": "mission of burma",
              "size": "small",
              "src": "thumbs-005.jpg",
            },
            Object {
              "id": 6,
              "name": "bass overload",
              "size": "tall",
              "src": "thumbs-006.jpg",
            },
            Object {
              "id": 7,
              "name": "folläkzoid",
              "size": "tall",
              "src": "thumbs-007.jpg",
            },
            Object {
              "id": 8,
              "name": "robot workshop",
              "size": "large",
              "src": "thumbs-008.jpg",
            },
            Object {
              "id": 9,
              "name": "angler fish",
              "size": "wide",
              "src": "thumbs-009.jpg",
            },
            Object {
              "id": 10,
              "name": "geometric bug",
              "size": "wide",
              "src": "thumbs-010.jpg",
            },
            Object {
              "id": 11,
              "name": "you mug",
              "size": "small",
              "src": "thumbs-011.jpg",
            },
            Object {
              "id": 12,
              "name": "mutant bmx",
              "size": "small",
              "src": "thumbs-012.jpg",
            },
            Object {
              "id": 13,
              "name": "nf fly",
              "size": "tall",
              "src": "thumbs-013.jpg",
            },
            Object {
              "id": 14,
              "name": "desert smoker",
              "size": "tall",
              "src": "thumbs-014.jpg",
            },
            Object {
              "id": 15,
              "name": "geometric butterfly",
              "size": "large",
              "src": "thumbs-015.jpg",
            },
            Object {
              "id": 16,
              "name": "dylan carlson",
              "size": "wide",
              "src": "thumbs-016.jpg",
            },
            Object {
              "id": 17,
              "name": "octopsss",
              "size": "wide",
              "src": "thumbs-017.jpg",
            },
            Object {
              "id": 18,
              "name": "collapse",
              "size": "small",
              "src": "thumbs-018.jpg",
            },
            Object {
              "id": 19,
              "name": "rat race",
              "size": "small",
              "src": "thumbs-019.jpg",
            },
            Object {
              "id": 20,
              "name": "papa m house",
              "size": "tall",
              "src": "thumbs-020.jpg",
            },
          ],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      null,
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "navClick": [Function],
          "navData": Array [
            Object {
              "active": false,
              "id": 0,
              "name": "gallery",
            },
            Object {
              "active": false,
              "id": 1,
              "name": "contact",
            },
            Object {
              "active": true,
              "id": 2,
              "name": "shop",
            },
          ],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Header
            navClick={[Function]}
            navData={
              Array [
                Object {
                  "active": false,
                  "id": 0,
                  "name": "gallery",
                },
                Object {
                  "active": false,
                  "id": 1,
                  "name": "contact",
                },
                Object {
                  "active": true,
                  "id": 2,
                  "name": "shop",
                },
              ]
            }
          />,
          <Content
            activeContent="shop"
            galleryData={
              Array [
                Object {
                  "id": 1,
                  "name": "leaf insect",
                  "size": "large",
                  "src": "thumbs-001.jpg",
                },
                Object {
                  "id": 2,
                  "name": "desert eyes",
                  "size": "wide",
                  "src": "thumbs-002.jpg",
                },
                Object {
                  "id": 3,
                  "name": "bardo pond",
                  "size": "wide",
                  "src": "thumbs-003.jpg",
                },
                Object {
                  "id": 4,
                  "name": "angled pike",
                  "size": "small",
                  "src": "thumbs-004.jpg",
                },
                Object {
                  "id": 5,
                  "name": "mission of burma",
                  "size": "small",
                  "src": "thumbs-005.jpg",
                },
                Object {
                  "id": 6,
                  "name": "bass overload",
                  "size": "tall",
                  "src": "thumbs-006.jpg",
                },
                Object {
                  "id": 7,
                  "name": "folläkzoid",
                  "size": "tall",
                  "src": "thumbs-007.jpg",
                },
                Object {
                  "id": 8,
                  "name": "robot workshop",
                  "size": "large",
                  "src": "thumbs-008.jpg",
                },
                Object {
                  "id": 9,
                  "name": "angler fish",
                  "size": "wide",
                  "src": "thumbs-009.jpg",
                },
                Object {
                  "id": 10,
                  "name": "geometric bug",
                  "size": "wide",
                  "src": "thumbs-010.jpg",
                },
                Object {
                  "id": 11,
                  "name": "you mug",
                  "size": "small",
                  "src": "thumbs-011.jpg",
                },
                Object {
                  "id": 12,
                  "name": "mutant bmx",
                  "size": "small",
                  "src": "thumbs-012.jpg",
                },
                Object {
                  "id": 13,
                  "name": "nf fly",
                  "size": "tall",
                  "src": "thumbs-013.jpg",
                },
                Object {
                  "id": 14,
                  "name": "desert smoker",
                  "size": "tall",
                  "src": "thumbs-014.jpg",
                },
                Object {
                  "id": 15,
                  "name": "geometric butterfly",
                  "size": "large",
                  "src": "thumbs-015.jpg",
                },
                Object {
                  "id": 16,
                  "name": "dylan carlson",
                  "size": "wide",
                  "src": "thumbs-016.jpg",
                },
                Object {
                  "id": 17,
                  "name": "octopsss",
                  "size": "wide",
                  "src": "thumbs-017.jpg",
                },
                Object {
                  "id": 18,
                  "name": "collapse",
                  "size": "small",
                  "src": "thumbs-018.jpg",
                },
                Object {
                  "id": 19,
                  "name": "rat race",
                  "size": "small",
                  "src": "thumbs-019.jpg",
                },
                Object {
                  "id": 20,
                  "name": "papa m house",
                  "size": "tall",
                  "src": "thumbs-020.jpg",
                },
              ]
            }
          />,
          null,
          <Footer
            navClick={[Function]}
            navData={
              Array [
                Object {
                  "active": false,
                  "id": 0,
                  "name": "gallery",
                },
                Object {
                  "active": false,
                  "id": 1,
                  "name": "contact",
                },
                Object {
                  "active": true,
                  "id": 2,
                  "name": "shop",
                },
              ]
            }
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "navClick": [Function],
            "navData": Array [
              Object {
                "active": false,
                "id": 0,
                "name": "gallery",
              },
              Object {
                "active": false,
                "id": 1,
                "name": "contact",
              },
              Object {
                "active": true,
                "id": 2,
                "name": "shop",
              },
            ],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "activeContent": "shop",
            "galleryData": Array [
              Object {
                "id": 1,
                "name": "leaf insect",
                "size": "large",
                "src": "thumbs-001.jpg",
              },
              Object {
                "id": 2,
                "name": "desert eyes",
                "size": "wide",
                "src": "thumbs-002.jpg",
              },
              Object {
                "id": 3,
                "name": "bardo pond",
                "size": "wide",
                "src": "thumbs-003.jpg",
              },
              Object {
                "id": 4,
                "name": "angled pike",
                "size": "small",
                "src": "thumbs-004.jpg",
              },
              Object {
                "id": 5,
                "name": "mission of burma",
                "size": "small",
                "src": "thumbs-005.jpg",
              },
              Object {
                "id": 6,
                "name": "bass overload",
                "size": "tall",
                "src": "thumbs-006.jpg",
              },
              Object {
                "id": 7,
                "name": "folläkzoid",
                "size": "tall",
                "src": "thumbs-007.jpg",
              },
              Object {
                "id": 8,
                "name": "robot workshop",
                "size": "large",
                "src": "thumbs-008.jpg",
              },
              Object {
                "id": 9,
                "name": "angler fish",
                "size": "wide",
                "src": "thumbs-009.jpg",
              },
              Object {
                "id": 10,
                "name": "geometric bug",
                "size": "wide",
                "src": "thumbs-010.jpg",
              },
              Object {
                "id": 11,
                "name": "you mug",
                "size": "small",
                "src": "thumbs-011.jpg",
              },
              Object {
                "id": 12,
                "name": "mutant bmx",
                "size": "small",
                "src": "thumbs-012.jpg",
              },
              Object {
                "id": 13,
                "name": "nf fly",
                "size": "tall",
                "src": "thumbs-013.jpg",
              },
              Object {
                "id": 14,
                "name": "desert smoker",
                "size": "tall",
                "src": "thumbs-014.jpg",
              },
              Object {
                "id": 15,
                "name": "geometric butterfly",
                "size": "large",
                "src": "thumbs-015.jpg",
              },
              Object {
                "id": 16,
                "name": "dylan carlson",
                "size": "wide",
                "src": "thumbs-016.jpg",
              },
              Object {
                "id": 17,
                "name": "octopsss",
                "size": "wide",
                "src": "thumbs-017.jpg",
              },
              Object {
                "id": 18,
                "name": "collapse",
                "size": "small",
                "src": "thumbs-018.jpg",
              },
              Object {
                "id": 19,
                "name": "rat race",
                "size": "small",
                "src": "thumbs-019.jpg",
              },
              Object {
                "id": 20,
                "name": "papa m house",
                "size": "tall",
                "src": "thumbs-020.jpg",
              },
            ],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        null,
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "navClick": [Function],
            "navData": Array [
              Object {
                "active": false,
                "id": 0,
                "name": "gallery",
              },
              Object {
                "active": false,
                "id": 1,
                "name": "contact",
              },
              Object {
                "active": true,
                "id": 2,
                "name": "shop",
              },
            ],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
